script.js

```diff
@@ -0,0 +1,6278 @@
+let img = new Image();
+let canvas = document.getElementById("canvas");
+let context = canvas.getContext('2d');
+context.imageSmoothingEnabled = false;
+let small_pic=[
+  ["00BCD4", "FFEB3B","FFEB3B","00BCD4"],
+  ["FFEB3B", "FFC107","FFC107","FFEB3B"],
+  ["FFEB3B", "FFC107","FFC107","FFEB3B"],
+  ["00BCD4", "FFEB3B","FFEB3B","00BCD4"]
+];
+let big_pic = [
```

Comment:
It's better to have such big data in a separate place, and got it with ajax request or import.
The same with small_pic.